Automattic/jetpack
# Introduce Jetpack Migration plugin

## Proposed changes:


Introduces a plugin named Jetpack Migration with the ability to connect the site, created with the `jetpack generate` tool but also this was added:

- The default connection screen as a start so we can test the functionality. (We can ignore the styling on the page now as this PR is to introduce the plugin into the repo)

![Screen Shot 2023-01-09 at 3 28 42 PM](https://user-images.githubusercontent.com/4074459/211258908-c39d2798-d21c-46e1-a8bf-777d31861837.png)


### Other information:


## Jetpack product discussion


For more details about this plugin, please see the WordPress.com Migration Plugin: pcmemI-1zN-p2
## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Launch a [JN site with this branch and plugin](https://jurassic.ninja/create/?jetpack-beta&branches.jetpack-migration=add/migration-plugin&wp-debug-log).
- Visit the Jetpack Migration menu
- Check and see if you can see the testing landing page for the plugin like the screenshot above
